### ISSUE TITLE
Installation help

### ISSUE THREAD

**Issue by gaje12** [tbarbanti/OpenFaceUE#1]
You do great work man!! I appreciate all your hard work. I'm having trouble installing. Is this a plugin I should build in a c++ project? Thanks!!!

**Comment by Moykul** [comment]
Hey Gaje12,

Any joy here?

**Comment by gaje12** [comment]
No. no joy. I've been focusing on building from scratch in spare time. Will definitely get a working copy up on the hub when its ready.
